core/src/test/java/org/apache/iceberg/TestSortOrder.java

```diff
@@ -365,4 +367,26 @@ public void testPreservingOrderSortedColumnNames() {
     Set<String> sortedCols = SortOrderUtil.orderPreservingSortedColumns(order);
     Assert.assertEquals(ImmutableSet.of("data"), sortedCols);
   }
+
+  @Test
+  public void testCaseSensitiveSortedColumnNames() {
```

Comment:
Not sure this test is testing any new code?  If we want an API level test, shouldnt it be for testing BaseReplaceSortOrder (which is the new code)?

Reply:
There are no tests currently for testing the caseSensitive method. Shall I remove it from here and add in a dfif pr?